test/e2e/cluster_logging_utils.go

```diff
@@ -19,51 +19,198 @@ package e2e
 import (
 	"errors"
 	"fmt"
+	"strconv"
+	"strings"
 	"time"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/api/resource"
+	api_v1 "k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
 const (
-	// ingestionTimeout is how long to keep retrying to wait for all the
-	// logs to be ingested.
-	ingestionTimeout = 10 * time.Minute
-	// ingestionRetryDelay is how long test should wait between
-	// two attempts to check for ingestion
-	ingestionRetryDelay = 25 * time.Second
+	// Duration of delay between any two attempts to check if all logs are ingested
+	ingestionRetryDelay = 10 * time.Second
 
-	synthLoggerPodName = "synthlogger"
+	// Assuming format is
+	// I1103 00:00:00.000000       0 logs_generator.go:67] <log_number> GET /api/v1/namespaces/ns/pods/XXX 000
+	logPrefixLength = 52
 
-	// expectedLinesCount is the number of log lines emitted (and checked) for each synthetic logging pod.
-	expectedLinesCount = 100
+	// Amount of requested cores for logging container in millicores
+	loggingContainerCpu = 10
```

Comment:
10 MB is enough, I checked

As for 10m, I think so, most time is spent on iowait. I'll check